app/src/main/java/com/willowtree/vocable/presets/PresetsRepository.kt

```diff
@@ -82,65 +82,48 @@ class PresetsRepository(context: Context) : KoinComponent {
         return database.categoryDao().getCategoryById(categoryId)
     }
 
-    suspend fun populateDatabase(numbersCategoryId: String, mySayingsCategoryId: String) {
+    suspend fun populateDatabase() {
```

Comment:
We may want to do some kind of check here to see if the presets updated. Otherwise users will be stuck on older versions.

Reply:
Hmm good call

Reply:
I think I might leave this as a separate task, handling preset scheme/version. To avoid bloating dis wan.